ably/realtime_conn_spec_test.go

```diff
@@ -1967,6 +1968,7 @@ func TestRealtimeConn_RTN14b(t *testing.T) {
 			Error:             bad,
 		}
 		var state ably.ConnectionStateChange
+		ablytest.Instantly.Recv(t, nil, change, t.Fatalf) // skip CONNECTING
```

Comment:
This is racy. To ensure we'll get the CONNECTING state into the `change` channel, the `c.Connect` must be below the `OnAll` listener registration.

Reply:
done 80ccec5874864feea4babdcc1db112cf30fdee3e